apps/web/app/(app)/environments/[environmentId]/surveys/[surveyId]/edit/components/SettingsView.tsx

```diff
@@ -28,20 +31,26 @@ export default function SettingsView({
   setLocalSurvey,
   actionClasses,
   attributeClasses,
+  userSegments,
   responseCount,
   membershipRole,
   colours,
+  isUserTargetingAllowed = false,
 }: SettingsViewProps) {
   return (
     <div className="mt-12 space-y-3 p-5">
       <HowToSendCard localSurvey={localSurvey} setLocalSurvey={setLocalSurvey} environment={environment} />
 
-      <WhoToSendCard
-        localSurvey={localSurvey}
-        setLocalSurvey={setLocalSurvey}
-        environmentId={environment.id}
-        attributeClasses={attributeClasses}
-      />
+      {isUserTargetingAllowed && (
```

Comment:
We should also show a card when `!isUserTargetingAllowed` to tell people what they are missing. We should tell them what's possibile with the feature; maybe show a skeleton of it and on self-hosting instances we should tell them that they need a valid enterprise license to use the feature and on formbricks cloud we need to tell them to upgrade to the user identification plan to use this feature.